src/database/seeds/0002-update-canonical_fields.sql

```diff
@@ -0,0 +1,188 @@
+DELETE FROM canonical_fields WHERE short_code = 'primary_revenue_growth_score';
+DELETE FROM canonical_fields WHERE short_code = 'program_categories';
+DELETE FROM canonical_fields WHERE short_code = 'program_expense_growth_score';
+DELETE FROM canonical_fields WHERE short_code = 'program_expense_ratio';
+DELETE FROM canonical_fields WHERE short_code = 'program_expense_score';
+DELETE FROM canonical_fields WHERE short_code = 'proposal_fiscal_sponsor_cantact_last_name';
+DELETE FROM canonical_fields WHERE short_code = 'rating_in_the_a&t_dimension';
+DELETE FROM canonical_fields WHERE short_code = 'rating_in_the_results_dimension';
+DELETE FROM canonical_fields WHERE short_code = 'rating_year';
+DELETE FROM canonical_fields WHERE short_code = 'rounded_cc_score_to_remove_decimals';
+DELETE FROM canonical_fields WHERE short_code = 'rounded_total_score_for_fa';
+DELETE FROM canonical_fields WHERE short_code = 'score_for_goals_score';
+DELETE FROM canonical_fields WHERE short_code = 'score_for_mission_statement';
+DELETE FROM canonical_fields WHERE short_code = 'score_for_vision_statement';
+DELETE FROM canonical_fields WHERE short_code = 'score_in_the_a&t_dimension';
+DELETE FROM canonical_fields WHERE short_code = 'score_in_the_financial_dimension';
+DELETE FROM canonical_fields WHERE short_code = 'score_in_the_results_dimension';
+DELETE FROM canonical_fields WHERE short_code = 'summary_of_the_activities';
+DELETE FROM canonical_fields WHERE short_code = 'text_of_the_donor_advisory_message_in_html';
+DELETE FROM canonical_fields WHERE short_code = 'the_arithmetic_for_the_cost_calculation';
+DELETE FROM canonical_fields WHERE short_code = 'total_accountability_score';
+DELETE FROM canonical_fields WHERE short_code = 'total_finace_score';
+DELETE FROM canonical_fields WHERE short_code = 'total_leadership_and_adaptablity_score';
+DELETE FROM canonical_fields WHERE short_code = 'total_points_avaiable_in_encompass';
+DELETE FROM canonical_fields WHERE short_code = 'web_location_of_proforma_990s_to_handle_pharmaceutical_valuation_adjustments';
+DELETE FROM canonical_fields WHERE short_code = 'what_is_the_overall_cc_score';
+DELETE FROM canonical_fields WHERE short_code = 'what_is_the_percent_of_their_independant_board';
+DELETE FROM canonical_fields WHERE short_code = 'when_was_this_overall_rating_created';
+DELETE FROM canonical_fields WHERE short_code = 'who_they_serve';
+DELETE FROM canonical_fields WHERE short_code = 'working_capital_ratio';
+DELETE FROM canonical_fields WHERE short_code = 'working_capital_score';
+DELETE FROM canonical_fields WHERE short_code = 'yes_no_indicator_this_rating_is_a_donor_advisory';
+
+-- Add new fields. Generated from 2023-04-19 base fields sheet
+INSERT INTO canonical_fields (label, short_code, data_type) VALUES
+('Organization Location', 'organization_location', '{ "type": "string" }' ),
+('Proposal Additional Contact Name', 'proposal_additional_contact_name', '{ "type": "string" }' ),
```

Comment:
We can encode multiple values for a single field; does this need to be separate from `proposal_primary_contact_name`?

Alternatively, do we also need the `proposal_additional_contact_{title,phone,email}` fields?

Reply:
Another good call. It is past time to figure out a way to encode these many-to-one and one-to-many relationships in our base fields spreadsheet.

Reply:
We've made some strides toward figuring out these relationships, I think the issues that capture them are likely https://github.com/PhilanthropyDataCommons/service/issues/396 and https://github.com/PhilanthropyDataCommons/service/issues/333.

Reply:
Created https://github.com/PhilanthropyDataCommons/service/issues/607 for this.